Sources_v3/StreamChatUI/ChatChannel/ChatMessageBubbleView.swift

```diff
@@ -233,11 +235,11 @@ private extension _ChatMessageGroupPart {
     var layoutOptions: LayoutOptions {
         var options: LayoutOptions = .all
         
-        if self.text.isEmpty {
+        if message.text.isEmpty {
```

Comment:
`swiftformat` removes `self` on commit, but swift unable to parse it otherwise.
Explicitly putting `message.text` solves issue